src/nft/pages/asset/Asset.css.ts

```diff
@@ -0,0 +1,119 @@
+import { style } from '@vanilla-extract/css'
+
+import { center, subhead } from '../../css/common.css'
+import { sprinkles, vars } from '../../css/sprinkles.css'
+
+export const image = style([
+  sprinkles({ borderRadius: '20', height: 'full', alignSelf: 'center' }),
+  {
+    width: 'calc(90vh - 165px)',
+    height: 'calc(90vh - 165px)',
+    maxHeight: '678px',
+    maxWidth: '678px',
+    boxShadow: `0px 20px 50px var(--shadow), 0px 10px 50px rgba(70, 115, 250, 0.2)`,
+    '@media': {
+      '(max-width: 1024px)': {
+        maxHeight: '64vh',
+        maxWidth: '64vh',
+      },
+      '(max-width: 640px)': {
+        maxHeight: '56vh',
+        maxWidth: '56vh',
+      },
+    },
+  },
+])
+
+export const container = style([
+  center,
+  {
+    minHeight: 'calc(100vh - 97px)',
+  },
+])
+
+export const marketplace = sprinkles({ borderRadius: '4' })
+
+export const tab = style([
+  subhead,
+  sprinkles({ color: 'darkGray', border: 'none', padding: '0', background: 'transparent' }),
+  {
+    selectors: {
+      '&[data-active="true"]': {
+        textDecoration: 'underline',
+        textDecorationColor: vars.color.genieBlue,
+        textUnderlineOffset: '8px',
+        textDecorationThickness: '2px',
+        color: vars.color.blackBlue,
+      },
+    },
+  },
+])
+
+export const creator = style({
+  '@media': {
+    '(max-width: 640px)': {
+      display: 'none',
+    },
+  },
+})
+
+export const columns = style([
+  sprinkles({
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    width: 'full',
+    paddingLeft: { sm: '16', lg: '24', xl: '52' },
```

Comment:
nit: suggest making a variable for these three

Reply:
I tried moving this into a variable just now and sprinkles gives an error.  It doesn't seem to recognize the sizings.  There's probably a way to do it, but I don't want to spend too much time figuring it out since we will likely rewrite soon

Reply:
all good